README.md

```diff
@@ -2,12 +2,47 @@
 
 ---
 
-> TechStack - React JS, Material UI
+## Installation
 
-_MORE INFO YET TO BE ADDED... STAY TUNED_
+The Open Event Frontend can be easily deployed on a variety of platforms, so just run the following commands when setting up for the first time:
 
-# DON'T JUST CLONE, PLEASE LEAVE A STAR, MOTIVATES US TO ADD MORE FEATURES TO THE PROJECT.
+- `yarn install or npm install  `
+- `yarn start or npm start `
+- Visit your app at [http://localhost:3000](http://localhost:3000).
 
-> NEW UI COMING UP , CHECK THIS SITE TO SEE THE LATEST UPDATES : [https://cowinvaccinetracker.stephinreji.me/](https://cowinvaccinetracker.stephinreji.me/)
+CoWIN Vaccination Slots Checking App is a website, mobile-ready,
+REACT JS-along with Material-UI
```

Comment:
Line 14: Does Not make any sense here, since you have added the same under tech header in line 29